Gemfile.lock

```diff
@@ -1,129 +1,218 @@
 GEM
   remote: https://rubygems.org/
   specs:
+    addressable (2.7.0)
```

Comment:
We should probably state somewhere what ruby, puppet version was used to generate the lock file; I guess we could call that the reference platform.

Like we discussed last time, there are so many combinations this was probably why it hadn't been included in the repo in the past.

Reply:
There is a default in the Gemfile that is used unless a specific version is given. This Gemfile.lock is generated from that default.